django_project_base/account/rest/account.py

```diff
@@ -35,7 +38,23 @@ class LoginViewSet(viewsets.ViewSet):
     )
     @action(detail=False, methods=['post'], url_name='login', authentication_classes=[], permission_classes=[])
     def login(self, request: Request) -> Response:
-        return login(request._request)
+        serializer_class = registration_settings.LOGIN_SERIALIZER_CLASS
```

Comment:
Tukaj, vidim, si vsko kodo skopiral iz one knjižnjice: je to sploh potrebnoi, glede na to, da cookieje brišeš v middleware-u? Tukaj bi teoretično samo moral postaviti sessionId JSON spremenljivko na sessionId (kar dobiš iz cookieja), če je zahtevani respons type == json. drugače pa ne bi bilo treba narest prav nič.